numba/tests/test_slices.py

```diff
@@ -185,6 +185,23 @@ def test_slice_indices_examples(self):
             str(e.exception)
         )
 
+    def test_slice_constant_lowering(self):
+        for ts in [slice(1, 3, None), slice(2, None, None), slice(None, 2, -1),
+                   slice(None, None, None), slice(None, None, -2)]:
```

Comment:
NOTE: I've checked that this hits all the branches in `make_slice_from_constant` as least once, but it doesn't test their combinatorial product.

Reply:
I've added all combinations of these inputs now.

Reply:
great, thanks.